packages/maskbook/src/plugins/VCent/Icons/icons.tsx

```diff
@@ -0,0 +1,36 @@
+import SvgIcon from "@material-ui/core/SvgIcon";
+import { ReactNode, forwardRef, memo } from "react";
+
+/**
+ * Create a icon from svg fragment
+ * @internal
+ * @param name Name of the Icon
+ * @param svg SVG content. Do not include <svg> tag
+ * @param viewBox The viewbox
+ * @param defaultSize Only use this when the icon is not square. Unit: px
+ * @returns A component that same type as icons from @material-ui/icons
+ */
+export function createIcon(
```

Comment:
Please don't copy `packages\icons\utils\index.tsx`. You can export this file and use it in `@dimensiondev/icons` package.

Reply:
This was because this non relative package import throw an error 'could not be found'. Although I am able to import other `@dimensiondev/ ` packages.  Is this driven by `tsconfig.json` ? but i presume not to alter this. Any ideas ?

Reply:
Add

```ts
export { createIcon } from './utils'
```

in `packages\icons\index.ts` and you can use it in `@dimensiondev/icons` package.

Reply:
hi. Cannot locate '@dimensiondev/icons'. This is all I can see. I will keep trying to figure this out. any more help greatly appreciated.
![image](https://user-images.githubusercontent.com/80292168/113318757-b0ca4700-9308-11eb-99f5-4f233ded81bc.png)